app/config/battle_logic.py

```diff
@@ -0,0 +1,23 @@
+class Battle:
+    def __init__(self, first,
+                 second
+                 ):

```

Comment:
All incoming variables will be on new line or on one line
`__init__(self, first, second):`

Add annotation for every incoming variables